pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -152,10 +152,12 @@ protected BrokerResponse handleRequest(long requestId, String query, @Nullable S
           break;
       }
     } catch (Exception e) {
-      LOGGER.info("Caught exception while compiling SQL request {}: {}, {}", requestId, query, e.getMessage());
+      LOGGER.info("Caught exception while compiling SQL request {}: {}, {}", requestId, query,
```

Comment:
How do we ensure the exception caught is compilation exception? Currently if it throws no access exception, this will cause NPE.

We probably need to explicitly check for exception type